subsys/bluetooth/controller/ll_sw/nordic/lll/lll_sync_iso.c

```diff
@@ -1146,6 +1236,10 @@ static void isr_rx_done(void *param)
 	}
 	lll->payload_tail = payload_index;
 
+	if (latency_event--) {
+		goto isr_rx_done_latency_event;
+	}
```

Comment:
What is the reasoning behind using `goto` here instead of a `while (latency_event--)`?

Reply:
it was a quick hack... will fix this